bin/Main.purs

```diff
@@ -108,8 +132,17 @@ main = launchAff_ do
       case cmd of
         GenerateOperators globs ->
           generateOperatorsCommand globs
-        FormatInPlace options numThreads globs -> do
-          operators <- readOperatorTable options.operators
+
+        GenerateRc cliOptions -> do
```

Comment:
I know that `rc` is used frequently to reference a config, but this might be more generally understandable if named `generate-config`. What do you think?